dali/imgcodec/decoders/jpeg/jpeg_mem.cc

```diff
@@ -518,10 +498,8 @@ uint8* UncompressLow(const void* srcdata, FewerArgsForCompiler* argball) {
 //  parameters won't get clobbered by the longjmp.  So we help
 //  it out a little.
 uint8* Uncompress(const void* srcdata, int datasize,
-                  const UncompressFlags& flags, int64* nwarn,
-                  std::function<uint8*(int, int, int)> allocate_output) {
-  FewerArgsForCompiler argball(datasize, flags, nwarn,
-                               std::move(allocate_output));
+                  const UncompressFlags& flags) {
+  FewerArgsForCompiler argball(datasize, flags);
```

Comment:
I guess now we could get rid of that, too.

Reply:
It is used to keep some state too, apparently. So I'll leave it for now.